packages/reporters/dev-server/src/templates/500.html

```diff
@@ -43,10 +43,10 @@
   <body>
     <div class="error-title-style">
       <div class="error-title-emoji">🚨</div>
-      <h1 class="error-title-heading"><%= error.message %></h1>
+      <h1 class="error-title-heading"><%- error.message %></h1>
     </div>
     <div class="error-stack-trace">
-      <%= error.stack %>
+      <%- error.stack %>
```

Comment:
Just went over the EJS docs again and `<%= error.stack %>` would escape html... changed it to `<%- error.stack %>` to render it properly